opentracing-spring-cloud/pom.xml

```diff
@@ -51,6 +51,26 @@
     </dependency>
 
     <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-jms</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.opentracing.contrib</groupId>
+      <artifactId>opentracing-jms-spring</artifactId>
```

Comment:
This instrumentation should not pull in anything that is not needed in user app. E.g. you might use this to just trace spring-web so we don't want to bring all JMS dependencies.